Solaris/EVS do not yet support security_groups so disable all
security_group quotas and remove the pie chart from the summary page.
This patch is not suitable for upstream contribution.

--- horizon-2015.1.2/horizon/templates/horizon/common/_limit_summary.html.orig	2015-12-12 12:53:10.612619487 -0500
+++ horizon-2015.1.2/horizon/templates/horizon/common/_limit_summary.html	2015-12-12 12:54:55.097278375 -0500
@@ -30,12 +30,6 @@
       </strong>
     </div>
 
-    <div class="d3_quota_bar">
-      <div class="d3_pie_chart_usage" data-used="{% quotapercent usage.limits.totalSecurityGroupsUsed usage.limits.maxSecurityGroups %}"></div>
-      <strong>{% trans "Security Groups" %} <br />
-        {% blocktrans with used=usage.limits.totalSecurityGroupsUsed|intcomma available=usage.limits.maxSecurityGroups|quotainf|intcomma %}Used <span> {{ used }} </span> of <span> {{ available  }} </span>{% endblocktrans %}
-      </strong>
-    </div>
   {% if usage.limits.totalVolumesUsed >= 0 %}
     <div class="d3_quota_bar">
       <div class="d3_pie_chart_usage" data-used="{% quotapercent usage.limits.totalVolumesUsed usage.limits.maxTotalVolumes %}"></div>

--- horizon-2015.1.2/openstack_dashboard/usage/quotas.py.orig	2015-12-12 12:56:35.439533474 -0500
+++ horizon-2015.1.2/openstack_dashboard/usage/quotas.py	2015-12-12 13:03:03.987504312 -0500
@@ -235,13 +235,11 @@
         # Remove the nova network quotas
         disabled_quotas.extend(['floating_ips', 'fixed_ips'])
 
-        if neutron.is_extension_supported(request, 'security-group'):
-            # If Neutron security group is supported, disable Nova quotas
-            disabled_quotas.extend(['security_groups', 'security_group_rules'])
-        else:
-            # If Nova security group is used, disable Neutron quotas
-            disabled_quotas.extend(['security_group', 'security_group_rule'])
-
+        # Solaris/EVS do not yet support security_groups, so disable all
+        # security_group quotas
+        disabled_quotas.extend(['security_groups', 'security_group_rules'])
+        disabled_quotas.extend(['security_group', 'security_group_rule'])
+        
         try:
             if not neutron.is_quotas_extension_supported(request):
                 disabled_quotas.extend(NEUTRON_QUOTA_FIELDS)
		 
